data/hai/hai reveal 3 early.txt

```diff
@@ -31,14 +31,14 @@ mission "Hai Building Mountaintop [1] Construction Fleet"
 			`When you return to Hai-home, you are directed to a government hangar where you are met by Xilin and Sayari, both looking very relieved.`
 			branch buildingdeferred
 				has "Building Mountaintop Deferred"
-			`	"I am very glad to see you returned, Captain <last>. Sayari said you were away on an important errand, but I've been hoping for a day now you would appear. I think you might be the solution to the current impasse."`
+			`	"I am very glad to see you returned, Captain <last>. Sayari said you were away on an important errand, but I've been hoping that any day now you would appear. I think you might be the solution to the current impasse."`
```

Comment:
I had thought that the player had been gone for some time now, which was why Xilin was hoping that "any day" the player would have come back instead of just a day.

Reply:
This one cropped up on another PR too - if he'd been hoping for you to appear for more than a day he'd have just sent you a message. He can do that, and would if it was an issue that spawned more than a day or two of consideration.

Reply:
We could always clear this up by changing it entirely to something more like "hoping since yesterday you would appear, and was even thinking of contacting you." or somesuch?

Reply:
Too many words, but "since yesterday" is possibly valid.